client/asset/eth/nodeclient.go

```diff
@@ -17,11 +17,12 @@ import (
 	"decred.org/dcrdex/dex"
 	dexeth "decred.org/dcrdex/dex/networks/eth"
 	"github.com/ethereum/go-ethereum"
-	"github.com/ethereum/go-ethereum/accounts"
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/state"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/crypto/secp256k1"
```

Comment:
Uh oh this is a cgo package.

Reply:
We should be able to recover the pubkey with our own secp256k1 packages, otherwise surely we can pull the pubkey from the keystore.

Reply:
Resolved in https://github.com/decred/dcrdex/pull/1358